packages/package_info_plus_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.1
```

Comment:
Hi @yash1200, I think we should not manually update the CHANGELOG.md on PRs, this will be done by the project maintainers on release.

In the https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md file we explain it better.

Reply:
Oh, didn't see that. I have reverted the package version.